### PR TITLE
Return recipient as an owner for the message inputs

### PR DESCRIPTION
**Pull request by xgreenx**
## Checklist
- [x] New behavior is reflected in tests

### CODE REVIEW

**Review by AurelienFT — APPROVED** [review]

**Review by netrome — APPROVED** [review]

**Comment by acerone85** [comment]
Is there an issue that explains what the bug is?

I had a look at the code and LGTM (won't approve as 1 codeowner approval should be required next per approval guidelines)

**Pull request merged**
